alert/topics.go

```diff
@@ -385,10 +389,13 @@ type bufHandler struct {
 	wg       sync.WaitGroup
 }
 
-func newHandler(h Handler) *bufHandler {
+func newHandler(h Handler, bufferSize int) *bufHandler {
+	if bufferSize == 0 {
```

Comment:
Why the difference in safety checks: above you use `<= 0`

Reply:
+1 -- this is actually a bug imho since `bufferSize` is an `int`.  this would panic 100% if someone passed in, say, -1.